datafusion/core/tests/physical_optimizer/enforce_distribution.rs

```diff
@@ -3208,10 +3215,11 @@ fn do_not_preserve_ordering_through_repartition() -> Result<()> {
 #[test]
 fn no_need_for_sort_after_filter() -> Result<()> {
     let schema = schema();
-    let sort_key = LexOrdering::new(vec![PhysicalSortExpr {
-        expr: col("c", &schema).unwrap(),
+    let sort_key: LexOrdering = [PhysicalSortExpr {
```

Comment:
FWIW another way to write this that i find sometimes clearer is:

```rst
    let sort_key =  LexOrdering::from([PhysicalSortExpr { ...}]);
```

It also often avoids an extra line.

Totally not needed for this PR -- I am just pointing it out